packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -172,7 +172,8 @@ class AnalysisServer {
 
     printTrace('dart ${command.skip(1).join(' ')}');
     _process = await processManager.start(command);
-    _process.exitCode.whenComplete(() => _process = null);
+    // Callback hookups' Futures are uninteresting.
+    _process.exitCode.whenComplete(() => _process = null); // ignore: unawaited_futures
```

Comment:
Why is it uninteresting?

In this case, it's uninteresting because it can't throw, I think.